app/assets/stylesheets/partials/style.scss

```diff
@@ -226,86 +207,81 @@ blockquote:hover {
 .footer {
   background: #f9f9f9;
   border-top: 1px solid #eee;
-  color: #777;
+  color: $gray-text;
   margin-top: 40px;
   padding: 20px 0 30px;
   text-shadow: 0 1px 0 #fff;
-}
-
-.footer.margin-top-0 {
-  margin-top: 0;
-}
 
-.footer .headline {
-  margin-bottom: 15px !important;
-}
+  .margin-top-0 {
+    margin-top: 0;
+  }
 
-.footer h1,
-.footer h2,
-.footer h3,
-.footer h4,
-.footer h5 {
-  border: none !important;
-  color: #555;
-  font-size: 18px;
-  font-weight: normal !important;
-  text-shadow: 0 1px 0 #fff;
-}
+  .headline {
+    margin-bottom: 15px;
+  }
 
-.footer p,
-.footer a {
-  font-size: 14px;
-}
-.footer p {
-  color: #777;
-}
+  h1,
+  h2,
+  h3,
+  h4,
+  h5 {
+    border: none;
+    color: #555;
+    font-size: 18px;
+    font-weight: normal;
+    text-shadow: 0 1px 0 #fff;
+  }
+  p {
+    font-size: 14px;
+    color: $gray-text;
+  }
 
-.footer a {
-  color: #72c02c;
-}
-.footer a:hover {
-  -moz-transition: all 0.4s ease-in-out;
-  -o-transition: all 0.4s ease-in-out;
-  -webkit-transition: all 0.4s ease-in-out;
-  color: #a8f85f;
-  transition: all 0.4s ease-in-out;
-}
+  a {
+    font-size: 14px;
+    color: $fsek-orange;
+  }
 
+  a:hover {
+    @include transition('all 0.4s ease-in-out');
+    color: $link-hover-color;
+  }
 
-/*Footer Subsribe*/
-.footer .footer-subsribe {
-  width: 85%;
-}
+  .footer-subscribe {
+    width: 85%;
+  }
 
-/*Blog Posts*/
-.footer .blog-list {
-  color: #dadada;
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
+  .blog-list {
+    color: #dadada;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    li {
+      margin-bottom: 7px;
+      padding: 0 0;

```

Comment:
Shorthand form for property `padding` should be written more concisely as `0` instead of `0 0`